intranet/apps/files/tests.py

```diff
@@ -0,0 +1,61 @@
+# -*- coding: utf-8 -*-
+
+from django.core.urlresolvers import reverse
+
+from unittest.mock import patch
+from ...test.ion_test import IonTestCase
+"""
+Tests for the filecenter.
+"""
+
+
+class FilesTest(IonTestCase):
+
+    @patch('django.contrib.messages.error')
+    @patch('stat.S_ISDIR')
+    @patch('intranet.apps.files.views.get_authinfo')
+    @patch('pysftp.Connection')
+    @patch('intranet.apps.files.models.Host')
```

Comment:
Rather then mocking out this, you should try and actually call the correct methods used in production.
Ideally, the only thing mocked out here should be pysftp, as it uses external resources.